src/sidebars/sidebars.json

```diff
@@ -215,6 +215,7 @@
             "handbook/company/story",
             "handbook/company/culture",
             "handbook/company/values",
+            "handbook/company/diversity",
```

Comment:
Should this be linked somewhere else (careers page)?

Reply:
added to careers page and to team page